readme.md

```diff
@@ -1,69 +1,257 @@
-* options.customer_email - (Required if customer_phone is not passed) Email of the customer
-* txref - Transaction Reference (Required) (Unique per transaction)
-* options.amount (if not passed, the Rave modal shows amount input)
-* options.onclose (function) - Set behaviour onclose of the modal
-* options.callback (function) - Set how you want to respond after payment is done
+### Installing
 
-The fields you pass here gets merged (with preference) on your app-wide setup done above.
-This final config is then passed Rave. See below more Rave options.
+The following steps will get you up and running.
 
-```
+1. Create a sample cordova project in your favorite directory
 
-PBFPubKey - (Required) Public key of the merchant
-amount - (Required) Amount to charge
-currency - (Optional, defaults to NGN) Currency to charge the card in
-country - (Optional, defaults to NG)
-customer_email - (Required if customer_phone is not passed) Email of the customer
-customer_firstname - (Required if customer_email is not passed) Phone number of the customer
-customer_lastname -(Optional) Full name of the customer
-pay_button_text - (Optional) Text to be displayed on the pay button
-custom_title - (Optional) Text to be displayed as the title of the payment modal
-custom_description - (Optional) Text to be displayed as a short modal description
-redirect_url - (Optional) URL to redirect to when transaction is completed
-custom_logo - (Optional) Link to the Logo image
-meta-[custom_param] - (Optional) Any other custom data you wish to pass (Without the square braces)
+    **NOTE:** You can skip this step if you already have a cordova project
+    ```
+    cordova create hello com.example.hello HelloWorld
+    ```
+    *This command will create the required directory structure for your cordova app. By default, the cordova create script generates a skeletal web-based application whose home page is the project's www/index.html file.*
+
+2. Test your sample project to be sure everything is working fine.
+
+    *We would be testing in the browser environment. To test on other environments check out the cordova documentation on [android](https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html) and [ios](https://cordova.apache.org/docs/en/latest/guide/platforms/ios/index.html).*
+    Run the following commands in order
+    ```
+    cd hello
+    cordova platform add browser
+    cordova run browser
+    ```
+    If everything works out. Life is good. **Moving On**
+
+3. Install Rave
+    - Navigate into your app directory (if you aren't already there). If you are using the same directory as this documentation, do ```cd hello```
+    - Run the following commands in order
+    ```
+    npm install cordova-rave
+    cd node_modules/cordova-rave && npm start 
+    ```
+    **NB: If you get a warning of outdated npm dependency/DOS issue, please [read this](https://github.com/impactmass/cordova-rave/issues/11#issuecomment-293965034)**
```

Comment:
@temiadesina this DOS issue has been fixed in some previous release. I've updated it in the readme here, but your fork still has it. You should update